tests/containers/containers_3rd_party.pm

```diff
@@ -64,8 +64,10 @@ sub run {
     my ($running_version, $sp, $host_distri) = get_os_release;
 
     # Define test images here
-    my @docker_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
-    my @podman_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
+    my $ex_reg        = get_var('REGISTRY', 'docker.io') . '/library';
+    my $os_reg        = 'registry.opensuse.org/opensuse';
```

Comment:
The registry is just `registry.opensuse.org`

Reply:
Same as above, to avoid this in lines below.